man/repart.d.xml

```diff
@@ -648,6 +648,27 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[Verity] Section Options</title>
+
+    <variablelist>
+      <varlistentry>
+        <term><varname>DataBlockSize=</varname></term>
```

Comment:
we generally want to suffix option names with their unit, i.e. here Bytes, i.e. hence VerityDataBlockSizeBytes=

Reply:
Noted. Will do once I know what the section naming will be.